iothub/device/src/ClientFactory.cs

```diff
@@ -19,7 +19,7 @@ internal class ClientFactory
 
         private static readonly TimeSpan s_regexTimeoutMilliseconds = TimeSpan.FromMilliseconds(500);
 
-        private static readonly Regex s_deviceIdParameterRegex = new Regex(
+        private static readonly Regex s_deviceIdParameterRegex = new(
```

Comment:
I know this is better, but this will never not look funny to me

Reply:
Not much different than `var foo = new Foo();` but it solves the problem of long lines in class field definitions, which I really appreciate the lack of redundancy. :)

But I hear you on it looking a bit off. ;)